keps/sig-network/1864-disable-lb-node-ports/README.md

```diff
@@ -136,6 +151,169 @@ re-enabling node port should not cause any traffic disruptions.
 Version skew from the control plane to kube-proxy should be trivial since kube-proxy's behavior is driven by the `nodePort` field
 and not the `allocateLoadBalancerNodePorts` field.
 
+## Production Readiness Review Questionnaire
+
+### Feature Enablement and Rollback
+
+_This section must be completed when targeting alpha to a release._
+
+* **How can this feature be enabled / disabled in a live cluster?**
+  - [X] Feature gate (also fill in values in `kep.yaml`)
+    - Feature gate name: ServiceLBNodePortControl
+    - Components depending on the feature gate: kube-apiserver
+  - [ ] Other
+    - Describe the mechanism:
+    - Will enabling / disabling the feature require downtime of the control
+      plane?
+    - Will enabling / disabling the feature require downtime or reprovisioning
+      of a node? (Do not assume `Dynamic Kubelet Config` feature is enabled).
+
+* **Does enabling the feature change any default behavior?**
+
+No, enabling the feature gate but not setting `spec.allocateLoadBalancerNodePorts` will not
+change any default behaviors in Service.
+
+* **Can the feature be disabled once it has been enabled (i.e. can we roll back
+  the enablement)?**
+
+  Yes, if the feature gate is disabled, new Services cannot use the new field, but existing Services
+  already using the field will continue to have it set.
+
+* **What happens if we reenable the feature if it was previously rolled back?**
+
+This field will be alpha feature gated for 1 release before being enabled by default, this
+ensures the API can safely handle rollback if the field has been set already.
+
+* **Are there any tests for feature enablement/disablement?**
+
+Yes, there will be unit tests for the Service API strategy which exercises the behavior
+with the feature gate enabled and disabled.
+
+### Rollout, Upgrade and Rollback Planning
+
+_This section must be completed when targeting beta graduation to a release._
+
+* **How can a rollout fail? Can it impact already running workloads?**
+
+* By default this should not impact any existing Services since we are not changing any default behaviors.
+* Enabling this feature on new clusters can impact workloads if load balancers depend on node ports without users
+being aware.
+
+* **What specific metrics should inform a rollback?**
+
+N/A. This feature does not have metrics.
```

Comment:
Do we have existing metrics around nodeport allocation? If so they should be mentioned here, as the counts will be different for these services.

Reply:
good point, will clarify this